src/config/global/prism.json

```diff
@@ -2,36 +2,120 @@
   "country": "Global",
   "alertFormActive": false,
   "icons": {
-    "rainfall": "icon_rain.png",
+    "rainfall_aggregation": "icon_rain.png",
+    "rainfall_anomaly": "icon_rain.png",
     "vegetation": "icon_veg.png",
     "temperature": "icon_climate.png",
-    "risk": "icon_impact.png",
+    "dry_spells": "icon_rain.png",
     "capacity": "icon_capacity.png",
     "tables": "icon_table.png"
   },
   "serversUrls": {
     "wms": [
-      "https://geonode.wfp.org/geoserver/prism/wms/",
       "https://odc.ovio.org/wms"
     ]
-  },   
+  },
   "map": {
     "latitude": 1.8,
     "longitude": 29.9,
     "zoom": 1.76
   },
-    "categories": {
-      "rainfall": {
-        "rainfall_amount": ["rainfall_dekad", "rainfall_agg_month"] ,
-        "rainfall_anomaly": ["rain_anomaly_dekad", "rain_anomaly_monthly"],
-        "dry_spells": ["streak_dry_days"],
-        "heavy_rain": ["days_heavy_rain", "streak_heavy_rain"]
-      },
-      "vegetation":{
-        "vegetation_conditions": ["ndvi_dekad", "ndvi_dekad_anomaly"]
-      },
-      "temperature": {
-        "land_surface_temperature": ["lst_daytime", "lst_anomaly", "lst_amplitude"]
-      }
+  "categories": {
```

Comment:
@wadhwamatic are these changes intentional?

Reply:
hmm.. sorry, the global changes should not have been here. let's remove those